README.md

```diff
@@ -10,6 +10,8 @@
 
 - A Momento API Key is required, you can generate one using
   the [Momento Console](https://console.gomomento.com)
+- A Momento cache is required, you can generate one by choosing the desired cloud provider and region using
+  the [Momento Console](https://console.gomomento.com/create)
```

Comment:
Since we only support AWS we don't really need the bit about choosing cloud provider.

```suggestion
- A Momento cache is required, you can generate one using
  the [Momento Console](https://console.gomomento.com/create)
```